files/zh-cn/learn/forms/form_validation/index.html

```diff
@@ -365,29 +365,7 @@ <h3 id="自定义错误信息">自定义错误信息</h3>
  <li>这依赖于他们使用的浏览器环境，意味着你可能在这种语言的页面里得到另一种语言的错误提示。</li>
 </ul>
 
-<table>
- <caption>在英文页面上的法语反馈信息版本</caption>
- <thead>
-  <tr>
-   <th scope="col">浏览器</th>
-   <th scope="col">渲染</th>
-  </tr>
- </thead>
- <tbody>
-  <tr>
-   <td>Firefox 17 (Windows 7)</td>
-   <td><img alt="Example of an error message with Firefox in French on an English page" src="/files/4329/error-firefox-win7.png" style="height: 97px; width: 228px;"></td>
-  </tr>
-  <tr>
-   <td>Chrome 22 (Windows 7)</td>
-   <td><img alt="Example of an error message with Chrome in French on an English page" src="/files/4327/error-chrome-win7.png" style="height: 96px; width: 261px;"></td>
-  </tr>
-  <tr>
-   <td>Opera 12.10 (Mac OSX)</td>
-   <td><img alt="Example of an error message with Opera in French on an English page" src="/files/4331/error-opera-macos.png" style="height: 83px; width: 218px;"></td>
-  </tr>
- </tbody>
-</table>
+<img alt="Example of an error message with Firefox in French on an English page" src="error-firefox-win7.png" style="height: 97px; width: 228px;">
```

Comment:
Please just remove the `style` attribute, yari will generate it by the original image
```suggestion
<img alt="Example of an error message with Firefox in French on an English page" src="error-firefox-win7.png">
```